python/nemo_relay/event_metadata.py

```
@@ -0,0 +1,36 @@
+    The registration applies to every subsequently published canonical Relay
+    Event until it is deregistered. Lower numeric priorities run first.
+    """
+    _register_event_metadata_injector(name, priority, injector)
+
+
+def deregister_injector(name: str) -> bool:
+    """Remove a global Event metadata injector by registration name."""
+    return _deregister_event_metadata_injector(name)
+
+
+__all__ = ["register_injector", "deregister_injector"]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)